homeassistant/components/homematicip_cloud/sensor.py

```diff
@@ -184,3 +191,35 @@ def state(self):
     def unit_of_measurement(self):
         """Return the unit this state is expressed in."""
         return 'lx'
+
+
+class HomematicipLightSensor(HomematicipIlluminanceSensor):
+    """Represenation of a HomematicIP Illuminance device."""
+
+    @property
+    def state(self):
+        """Return the state."""
+        return self._device.averageIllumination
+
+
+class HomematicipPowerSensor(HomematicipGenericDevice):
+    """Represenation of a HomematicIP power measuring device."""
+
+    def __init__(self, home, device):
+        """Initialize the  device."""
+        super().__init__(home, device, 'Power')
+
+    @property
+    def device_class(self):
+        """Return the device class of the sensor."""
+        return 'power'
```

Comment:
Please only use device constants as defined in `sensor/__init__.py`

Reply:
Hi,
then this PR depends on architectur issue [Additional device_class power](https://github.com/home-assistant/architecture/issues/140),
because there is currently no matching sensor type.

My commit [d191287](https://github.com/home-assistant/home-assistant/pull/20747/commits/d191287faaa0f8eb1dcca4c49539d99981c10663) could be reverted, if a device_class_power exists.

BR
Markus

Reply:
device class is optional.

Reply:
Thanks a lot. Ready the docs might help next time.